app/models/history.rb

```diff
@@ -463,30 +466,42 @@ def self.clear_follow_up_flag(history)
   end
 
   private_class_method def self.compose_message(history, field)
-    creator = history[:household_status] == :patient ? 'User' : 'System'
+    creator = history[:initiator_id].nil? ? 'System' : 'User'
     verb = field[:new_value].blank? ? 'cleared' : 'changed'
     from_text = field[:old_value].blank? ? 'blank' : "\"#{field[:old_value]}\""
     to_text = field[:new_value].blank? ? 'blank' : "\"#{field[:new_value]}\""
 
     comment = "#{creator} #{verb} #{field[:name]} from #{from_text} to #{to_text}"
-    comment += compose_explanation(history, field)
+    comment += compose_explanation(history)
     comment += history[:note] unless history[:note].blank?
     comment += '.'
     comment += " Reason: #{history[:reason]}" unless history[:reason].blank?
     comment
   end
 
-  private_class_method def self.compose_explanation(history, field)
-    if history[:household_status] == :patient && history[:propagation] == :group
-      " and chose to update this #{field[:type]} for all household members"
-    elsif history[:household_status] == :patient && history[:propagation] == :group_cm
-      " and chose to update this #{field[:type]} for household members under continuous exposure"
-    elsif history[:household_status] != :patient && history[:propagation] == :group
-      " because User updated #{field[:name]} for another member in this monitoree's household and chose to update this
-        #{field[:type].nil? ? 'field' : field[:type]} for all household members"
-    elsif history[:household_status] != :patient && history[:propagation] == :group_cm
-      " because User updated #{field[:name]} for another member in this monitoree's household and chose to update this
-        #{field[:type].nil? ? 'field' : field[:type]} for household members under continuous exposure"
+  private_class_method def self.compose_explanation(history)
```

Comment:
When making a change from the `Monitoring Actions` tab, and applying that change to selected household members, there is no indication on the initiator of the change that selected household members also had those changes applied. I think I would have expected there to be some sort of message there. Is this something that the PO team had input on?

Reply:
There wasn't anything like that in the mock-ups, but I agree that would make sense and I can certainly add it.

Reply:
I think if it wasn't in the mock-ups, we can ignore me on this for now. I don't want to expand the scope unnecessarily.